Resources/Prototypes/Recipes/Lathes/cooking.yml

```diff
@@ -104,3 +104,10 @@
   completetime: 0.4
   materials:
     Steel: 100
+
+- type: latheRecipe
+  id: FoodPlateMuffinTin
+  result: FoodPlateMuffinTin
+  completetime: 0.4
+  materials:
+    Steel: 50
```

Comment:
Move this above FoodKebabSkewer so it is below the FoodPlateTin recipe.
You will also need to add this to lathe.yml so the recipe can be used.
Reminder: No web edits, please test your changes in-game.